src/content/configuration/output.mdx

```diff
@@ -1135,6 +1135,7 @@ module.exports = {
   //...
   output: {
     // One of the below
+    publicPath: 'auto', // It automatically determines the publicPath from <script> tag. Only works with web and web-worker targets.
```

Comment:
`publicPath` can be derived in many ways more than `<script />` when set to `auto`. Here's the pull request introducing `auto` value for `publicPath`. https://github.com/webpack/webpack/pull/11258